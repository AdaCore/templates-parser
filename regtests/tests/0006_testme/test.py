from test_support import *

run('testme', ["testme7.tmplt"])
run('print_tree', ["testme7.tmplt"])
