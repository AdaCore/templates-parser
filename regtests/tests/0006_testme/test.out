
1
[one]
2
[two]
3
[three]
1
[(one)]
2
[(two)]
3
[]

1
[one]
2
[two]
3
[three]
1
[(one)]
2
[(two)]

>one
>(one)
>two
>(two)
>three
>

>one
>(one)
>two
>(two)

[INFO] testme7.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[TABLE]
  [BLOCK]
    [COMMON]
      [TABLE]
        [BLOCK]
          [COMMON]
            [TEXT] @_TABLE_LINE_@
            [TEXT] [@_MATDSIZE_@]
        [END_BLOCK]
      [END_TABLE]
  [END_BLOCK]
[END_TABLE]
[TEXT] 
[TABLE]
  [BLOCK]
    [COMMON]
      [TABLE]
        [BLOCK]
          [COMMON]
            [IF] @_EXIST:MATDSIZE_@
              [TEXT] @_TABLE_LINE_@
              [TEXT] [@_MATDSIZE_@]
            [END_IF]
        [END_BLOCK]
      [END_TABLE]
  [END_BLOCK]
[END_TABLE]
[TEXT] 
[TABLE]
  [BLOCK]
    [COMMON]
      [TEXT] >@_V5_@
      [TEXT] >@_V6_@
  [END_BLOCK]
[END_TABLE]
[TEXT] 
[TABLE]
  [BLOCK]
    [COMMON]
      [IF] @_EXIST:V6_@
        [TEXT] >@_V5_@
        [TEXT] >@_V6_@
      [END_IF]
  [END_BLOCK]
[END_TABLE]
